samples/word/99-preview-apis/manage-fields.yaml

```diff
@@ -0,0 +1,120 @@
+id: word-manage-fields
+name: Get fields
+description: This sample shows how to get existing fields.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#run").click(() => tryCatch(run));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function run() {
+          await Word.run(async (context) => {
+            let field = context.document.body.fields.getFirstOrNullObject();
+            field.load();
+
+            await context.sync();
+
+            console.log("Code: " + field.code);
+            console.log("Result: " + field.result);
+
+            let parentBody = field.parentBody;
+            parentBody.load("text");
+            await context.sync();
+
+            console.log("Text of parentBody: " + JSON.stringify(parentBody.text));
+          });
+        }
+
+        async function setup() {
+          await Word.run(async (context) => {
+            context.document.body.clear();
+            context.document.body.insertParagraph("One more paragraph. ", "Start");
+            context.document.body.insertParagraph("Inserting another paragraph. ", "Start");
+            context.document.body.insertParagraph(
+              "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
+              "Start"
+            );
+            context.document.body.paragraphs
+              .getLast()
+              .insertText(
+                "To make your document look professionally produced, Word provides header, footer, cover page, and text box designs that complement each other. For example, you can add a matching cover page, header, and sidebar. Click Insert and then choose the elements you want from the different galleries. ",
+                "Replace"
+              );
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample demonstrates how to get fields, which are placeholders for dynamic data in your document. To learn
+                more
+                about fields, refer to <a
+                    href="https://support.microsoft.com/office/c429bbb0-8669-48a7-bd24-bab6ba6b06bb">Insert,
```

Comment:
Not a problem for the sample, but it's really unclear how to return to the Run task pane after clicking this.